internal/app/cli.go

```diff
@@ -48,6 +48,44 @@ func CreateClusterCommand(p PolicyAutomation) *cli.Command {
 		Name:  "cluster",
 		Usage: "Manage policies against GKE clusters",
 		Subcommands: []*cli.Command{
+			{
+				Name:  "json-data",
+				Usage: "Print cluster api raw json data",
+				Flags: append([]cli.Flag{
+					&cli.StringFlag{
```

Comment:
As exactly same set of flags repeat in `CreateClusterCommand` I suggest to extract that to separate function (example - flags for policy sources in `getPolicySourceFlags`)